util/files_test.go

```diff
@@ -0,0 +1,55 @@
+package util
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFiles(t *testing.T) {
+	rootPath, err := os.MkdirTemp("", "slog-agent-test-util-*")
```

Comment:
there is a `t.TempDir()` helper function, might be convenient to use since it doesn't require `defer os.RemoveAll()`

Reply:
replaced all MkdirTemp with TempDir